detektConfig.yml

```diff
@@ -1,3 +1,7 @@
+complexity:
+  TooManyFunctions:
```

Comment:
Is it really necessary 20? Can you reduce it at, at least, 10?

Reply:
Is it ok for you if i remove this setting in the next PR?